.github/workflows/build_and_test.yml

```diff
@@ -54,7 +66,7 @@ jobs:
           }
           & $python_exec -m pip install -U pytest pywin32
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
This can be updated to use the action wrapped in [eprosima-CI](https://github.com/eProsima/eProsima-CI/tree/main) repository. Thus, updating the external action would be enough to upgrade every eProsima workflow.

Reply:
Done in 2899e3427dbebfc5021439914e7b2e6b16aea818